docker/driver-loader-legacy/docker-entrypoint.sh

```diff
@@ -26,4 +26,5 @@ do
     ln -s "$i" "/usr/src/$base"
 done
 
-/usr/bin/falco-driver-loader "$@"
+/usr/bin/falcoctl driver config "$@"
```

Comment:
This will need some more work, i am not sure what we want to do.

Reply:
Same goes for other occurrences.

Reply:
> This will need some more work, i am not sure what we want to do.

What do you mean? what do we miss here?

Reply:
Oh the parameters being passed to falco-driver-loader were surely different from the one being passed to `falcoctl driver`; also `config` might be wrong there, since we may want to let users decide which subcmd to invoke.

Reply:
We decided to reimplement a similar switch logic to the one supported by falco-driver-loader (https://github.com/falcosecurity/falco/blob/master/scripts/falco-driver-loader#L752) managing each flag.